windows/src/engine/keyman32/syskbdnt64.cpp

```diff
@@ -31,6 +31,9 @@
 ////////////////////////////////////////////////////////////////////////////
 
 #include "pch.h"
+
+#ifndef _WIN64
```

Comment:
Shouldn't this be the other way round?

```suggestion
#ifdef _WIN64
```

Reply:
No, this is correct as it is -- even though that is confusing! The syskbd* files are only used in the x86 host.

Reply:
Can we add a comment or fill in the description at the top?

Reply:
> Can we add a comment or fill in the description at the top?

See #11929. Apologies for missing this before merge.